tests/integrationv2/test_key_update.py

```diff
@@ -28,8 +28,8 @@ def test_s2n_server_key_update(managed_process, cipher, provider, other_provider
     port = next(available_ports)
 
     update_requested = b"K"
-    server_data = data_bytes(10)
-    client_data = data_bytes(10)
+    server_data = "server data"
+    client_data = "client data"
```

Comment:
I think these are too generic. I could easily see them appearing in debug messaging. Could you still append the data_bytes to them? Or otherwise make them somehow random / unique?

Reply:
I made them more fun 🤷‍♀️